cou/steps/execute.py

```diff
@@ -110,4 +127,4 @@ async def apply_plan(plan: UpgradeStep, interactive: bool) -> None:
             case "s":
                 logger.info("Skipped")
             case _:
-                logger.info("No valid input provided!")
+                logger.debug("No valid input provided!")
```

Comment:
I think that it's not necessary, because it's obvious if I do
```bash
Upgrade plan for 'octavia-dashboard' to wallaby (c)ontinue/(a)bort/(s)kip:test1
Upgrade plan for 'octavia-dashboard' to wallaby (c)ontinue/(a)bort/(s)kip:test2
Upgrade plan for 'octavia-dashboard' to wallaby (c)ontinue/(a)bort/(s)kip:c
```
to see that I provided wrong input. Your suggestion will be
```bash
Upgrade plan for 'octavia-dashboard' to wallaby (c)ontinue/(a)bort/(s)kip:test1
No valid input provided!
Upgrade plan for 'octavia-dashboard' to wallaby (c)ontinue/(a)bort/(s)kip:test2
No valid input provided!
Upgrade plan for 'octavia-dashboard' to wallaby (c)ontinue/(a)bort/(s)kip:c
```
Personally I do not like it.

Reply:
As discussed elsewhere, I still think it'd better to provide error message to user to avoid confusion.